components/server/ee/src/container-module.ts

```diff
@@ -46,8 +46,6 @@ import { ChargebeeService } from "./user/chargebee-service";
 import { StripeService } from "./user/stripe-service";
 import { EligibilityService } from "./user/eligibility-service";
 import { AccountStatementProvider } from "./user/account-statement-provider";
-import { WorkspaceStarterEE } from "./workspace/workspace-starter";
```

Comment:
WorkspaceStarterEE does no longer container any logic, so I deleted it and also removed the corresponding bindings.